### ISSUE TITLE
CLI: Class regex does not allow for numbers in class name

### ISSUE THREAD

**Issue by klauer** [pcdshub/typhos#373]
https://github.com/pcdshub/typhos/blob/df26dd62b3b1156783388ad9fa12d9c3371eca16/typhos/cli.py#L133

It seems like this could be simplified to not even use regular expressions. 

It definitely won't accept numbers in the class name; but what else? 
Are there any issues with brackets and such in the dictionary portion? Nesting?

**Comment by ZLLentz** [comment]
The goal of this regex is to allow us to differentiate between `happi` names and full class instantiation instructions, right?
e.g. both of these should work:
```
typhos im3l0
typhos "pcdsdevices.pim.PPM[{'prefix': 'IM3L0:PPM', 'name': 'im3l0'}]"
```
If so, then the best way to resolve this is to stop supporting multiple inputs to the same parameter, and instead do something like:
```
typhos --happi im3l0
typhos --class pcdsdevices.pim.PPM --kwargs prefix=IM3L0:PPM name=im3l0
```

**Comment by klauer** [comment]
Yeah, that's the intent of it.

Your suggestion is without question a much more user-friendly and less ambiguous than how it is now.
The only thing that's not immediately clear is how that would support loading multiple class entries:

```
typhos --happi im3l0 at2l0 ...   # good
typhos --class class1 --kwargs prefix=IM3L0:PPM --class class2 --kwargs prefix=IM3L0:PPM2  #  ?
```

**Comment by ZLLentz** [comment]
Ugh, users and their need to see TWO devices at the same time.

I agree that the arg passing and parsing here is extremely awkward. You can sort of do this in argparse with `action=append` and `nargs=+` in the kwargs arg then in the code we get something like:
```
args.class == ['class1', 'class2']
args.kwargs == [['prefix=IM3L0:PPM'], ['prefix='IM3L0:PPM2']]
```
Not necessarily the greatest, but workable.

**Comment by ZLLentz** [comment]
Maybe just supporting this would be easiest:
```
typhos --class pcdsdevices.pim.PPM IM3L0:PPM name=im3l0 --class pcdsdevices.pim.PPM IM4L0:PPM name=im4l0 --happi at2l0
```

**Comment by klauer** [comment]
Yeah, that seems pretty nice. `--class class prefix [kwargs [kwargs...]]` 👍
